npm/config/config.go

```diff
@@ -16,8 +17,9 @@ var DefaultConfig = Config{
 	ListeningAddress: "0.0.0.0",
 
 	Transport: GrpcServerConfig{
-		Address: "0.0.0.0",
-		Port:    defaultGrpcPort,
+		Address:    "0.0.0.0",
+		Port:       defaultGrpcPort,
```

Comment:
maybe `ServerPort` to be extra explicit alongside the `RemotePort`?

Reply:
ServicePort maybe?